generated/nidcpower/nidcpower/session.py

```diff
@@ -5157,6 +5187,138 @@ def create_advanced_sequence(self, sequence_name, property_names, set_as_active_
+
+            reset (bool): Specifies whether to reset channel(s) during the initialization procedure.
+
+            option_string (dict): Specifies the initial value of certain properties for the session.
+                The syntax for **optionString** is a list of properties with an assigned value where
+                1 is True and 0 is False. For example:
+
+                Simulate=0, DriverSetup=Model:<model number>; BoardType:<bus connector>
+
+                To simulate a multi-instrument session when independent_channels is True, set
+                Simulate to 1 and list multiple instruments for DriverSetup. For example:
+
+                Simulate=1, DriverSetup=ResourceName:<instrument name>; Model:<model number>;
+                BoardType:<bus connector> & ResourceName:<resource name>; Model:<model number>;
+                BoardType:<bus connector>
+
+                You do not have to specify a value for all the properties. If you do not specify a
+                value for a property, the default value is used.
+
+            independent_channels (bool): Specifies whether to initialize the session with
+                independent channels. Set this argument to False on legacy applications or if you
+                are unable to upgrade your NI-DCPower driver runtime to 20.6 or higher.
+
+
+        Returns:
+            vi (int): Returns a session handle that you use to identify the device in all
+                subsequent NI-DCPower method calls.
+
+        '''
+        if independent_channels:
+            resource_string = resource_name  # store potential modifications to resource_name in a separate variable
+
+            if channels:
+                # if we have a channels arg, we need to try and combine it with the resource name
+                # before calling into initialize with independent channels
+                channel_list = (resource_name + "/" + channel for channel in channels.split(","))
+                resource_string = ",".join(channel_list)
+
+                import warnings
+                warnings.warn(
+                    "Attempting to initialize an independent channels session with a channels argument. The resource "
+                    "name '" + resource_name + "' will be combined with the channels '" + channels + "' to form the "
+                    "fully-qualified channel list '" + resource_string + "'. To avoid this warning, use a "
+                    "fully-qualified channel list as the resource name instead of providing a channels argument.",
+                    DeprecationWarning
+                )
+
+                if "," in resource_name:
+                    raise ValueError(
+                        "Channels can't be combined with multiple devices in the resource name '" + resource_name + "'. "
+                        "Use a single device in the resource name or provide a list of fully-qualified channels as the "
```

Comment:
`Use a single device in the resource name` - I think we can skip this part and directly say pass FQCN in `resource name`.

Reply:
https://github.com/ni/nimi-python/pull/1594/commits/171999237b561b76ffd6432fe286c8134fbd8742